src/seedu/addressbook/ui/TextUi.java

```diff
@@ -16,19 +16,12 @@
  * Text UI of the application.
  */
 public class TextUi {
-
-    /** A decorative prefix added to the beginning of lines printed by AddressBook */
-    private static final String LINE_PREFIX = "|| ";
-
-    /** A platform independent line separator. */
-    private static final String LS = System.lineSeparator();
-
+    
```

Comment:
more formatting related stuff (like the divider below) can be moved to the formatter class?